documentation/commands/run.md

```diff
@@ -2,15 +2,19 @@
 
 _Runs an Exosphere application on the local machine_
 
-Usage: `exo run`
+Usage: `exo run [flags]`
 
-- dockerizes all services and their dependencies (databases),
+- dockerizes all services and their dependencies,
   so no installation of programming languages or runtimes is necessary.
 - prepares them (installing dependencies, compiling)
-- boots the dependencies up first, so that services see all dependencies running
-- monitors for file system changes and reboots the affected services
+- reboots services when they go down
+- `exo generate docker-compose` is run at the beginning of this command
+
+Flags:
+- `--production`   Run in production mode
```

Comment:
I think this should mention uses the production dockerfiles. "Production mode" isn't clear about what it is referring to